cypress/support/page-objects/ab-testing.po.ts

```diff
@@ -0,0 +1,19 @@
+export class ABTestPage {
+  static PAGE_URL = 'abtest';
+  static HOME_LINK_TEXT = 'A/B Testing';
+
+  static HEADER_TEXT = 'A/B Test';
+  static BODY_TEXT = 'Also known as split testing';
+
+  static visit() {
+    cy.visit(this.PAGE_URL);
```

Comment:
Should this cy action not return from this function as well, in the case it would need to be chained off of in the future?

Same for all .po.ts files visit() functions

Reply:
Done